### ISSUE TITLE
val loss up

### ISSUE THREAD

**Issue by lijain** [xingyizhou/CenterNet#752]
My training setup is:  python src/main.py ctdet ----arch res_101 --exp_id dataset --batch_size 24 --master_batch 6 --lr 5e-4  --gpus 0,1,2,3  --lr_step 90,120 --num_epochs 140 
imagesize=[800,800]
I would like to ask, the best model I am in is 65 epoch. I checked the next log and found that after 70 epoch, the loss has been declining slowly, but valloss is no longer falling and has a tendency to fit. I want to ask why??? Is my LR setting too large?
